### ISSUE TITLE
Ability to edit comments

### ISSUE THREAD

**Issue by JohnstonCode** [CodeHubApp/CodeHub#377]
Hi not sure if the github api would allow this but the ability to edit comments would be useful

**Comment by splorp** [comment]
Agreed. Because of our workflow, we use comments to track certain criteria and changes made to issues. This would be very handy via the app.

**Comment by vinyldarkscratch** [comment]
+1 — just ran into this issue myself.
